pkg/resources/table.go

```diff
@@ -139,6 +139,12 @@ var tableSchema = map[string]*schema.Schema{
 					Default:     "",
 					Description: "Masking policy to apply on column. It has to be a fully qualified name.",
 				},
+				"collate": {
```

Comment:
We generate the docs automatically from the schema and we push it as part of the change. You can run make-pre push to generate them. Also, it would be nice to add an example with collate to examples/resources/snowflake_table/resource.tf

Reply:
Thank you !
I fixed golangci and doc-check and added example !

https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2496/commits/9f2fb0f414887a2d437f2e1e19bb52faf0ef2e26
https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2496/commits/1ba35b07682bc2fb89b71c33eaebff9e36b8eeea